### PR TITLE
Enhance UI of Shifting Board

### LINKED ISSUE #6636: Design Enhancement Scope in Shifting Board
**Describe the bug**

Currently, the shifting board has three scroll bars and it doesn't feel good, when we use it

**To Reproduce**

1. Go to shifting board
2. See error

**Expected behavior**

An effective way to utilize the board and make the user experience better

Note: Whoever wishes to work on this issue, get the mock-up design approved in the EOD call, then only, the issue will be assigned to the respective author.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Arzzam**
## Proposed Changes

- Fixes #6636
- Fix the height of the container and fix the position of the headers in the shifting board
- Add Buttons and update the left and right scroll using that icon clicks 

https://github.com/coronasafe/care_fe/assets/92790024/185f5097-cf3b-47ba-99d2-1bd01305de9c

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by Pranshu1902** [comment]
@Arzzam there is still a scroll bar on the right side of the page (shown in blue in the image below)
Screenshot from 2024-01-01 15-45-56

And could you add a transparent circular effect on hover around the arrow? Kind of like the export button on the same page
image
Reference:
image

cc @rithviknishad

**Comment by Arzzam** [comment]
Yeah sure, I will add some circular effect for the buttons.
How can I fix the scroll bar. Can I proceed with a App router change?

Is these buttons are ok in hover state? 
image

**Comment by Pranshu1902** [comment]
@rithviknishad could you confirm the method to remove the extra scroll bar on the extreme right of the page

**Comment by nihal467** [comment]
@Arzzam 

image

- [ ] Keep the left and right arrow outside the board, where I marked the red circle
- [ ] If we are in the leftmost and there is no more left to go with the arrow, keep the left arrow hidden similarly to the rightmost as well

**Comment by Arzzam** [comment]
@nihal467 
https://github.com/coronasafe/care_fe/issues/6636#issuecomment-1858796850

As I discussed with @Pranshu1902 in issue, It was the one he came up with
so I done just like he said

**Comment by nihal467** [comment]
@Arzzam The current UI is overlapping with the shifting board data, and it doesn't look good. Please modify it based on the comment i made above

**Comment by Arzzam** [comment]
Hi @Pranshu1902, @nihal467,

I have changed it as requested. Review and Merge it.

https://github.com/coronasafe/care_fe/pull/6956#issuecomment-1873279199 
If this change requires, Kindly create a new issue and I will work on that.

Thank you ✌🏻.

**Comment by nihal467** [comment]
@Arzzam When we are on the leftmost side and there is no more left to go, as per the comment, we need to hide the left arrow button, not grey it out. Similar behavior should be applied to the right side as well.

image

**Comment by Arzzam** [comment]
Updated as requested @Pranshu1902 @nihal467 @rithviknishad

File: src/Components/Shifting/BoardView.tsx [hunk 1]
```diff
@@ -57,9 +57,79 @@ export default function BoardView() {
...
+            icon={`l-arrow-${direction}`}
+            className="absolute inset-y-0 left-0 z-10 h-10 w-10 cursor-pointer hover:opacity-100"
+            onClick={() => handleOnClick(direction)}
+          />
+        </div>
+      )
+    );
+  };
+
+  console.log("left", isLeftScrollable, "right", isRightScrollable);
```
Comment (Ashesh3) [inline src/Components/Shifting/BoardView.tsx]:
```suggestion
```

**Review by Ashesh3 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

File: src/Components/Shifting/BoardView.tsx [hunk 2]
```diff
@@ -57,9 +57,78 @@ export default function BoardView() {
...
+          <CareIcon
+            icon={`l-arrow-${direction}`}
+            className="absolute inset-y-0 left-0 z-10 h-10 w-10 cursor-pointer hover:opacity-100"
+            onClick={() => handleOnClick(direction)}
+          />
+        </div>
+      )
+    );
+  };
+
```
Comment (Ashesh3) [inline src/Components/Shifting/BoardView.tsx:128]:
```suggestion
```

**Review by Pranshu1902 — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**